frontend/public/locales/en/translation.json

```diff
@@ -168,7 +168,7 @@
     "bareMetalAsset.emptyState.subtitle": "Click <bold>Create bare metal asset</bold> or <bold>Import bare metal assets</bold> to create your resource.",
     "bareMetalAsset.emptyState.title": "You don't have any bare metal assets",
     "bareMetalAsset.importAction.button": "Open file",
-    "bareMetalAsset.importAction.message": "The first line of the CSV file must be a comma separated header row that defines the following columns: hostName, hostNamespace, bmcAddress, macAddress, role (optional), username, password.",
+    "bareMetalAsset.importAction.message": "The first line of the CSV file must be a comma separated header row that defines the following columns: hostName, hostNamespace, bmcAddress, macAddress, role Optional:, username, password.",
```

Comment:
"The first line of the CSV file must be a comma separated header row that defines the following columns: hostName, hostNamespace, bmcAddress, macAddress, role. Optional: you can define username, password." 

(Note the change to two complete sentences, some way to get Optional: out of the middle?)

Could it be? "Optional columns to define: username, password."

Reply:
@Randy424 one comment above in that confusing sentence, otherwise I approved.

Reply:
@swopebe 
I think "(optional)" here, was referring to "role". Can we keep it as **_"...hostName, hostNamespace, bmcAddress, macAddress, role (optional), username, password."_**?

Reply:
Why was this merged like this without resolving?